test/jdk/com/sun/security/auth/module/AllPlatforms.java

```diff
@@ -21,44 +21,39 @@
  * questions.
  */
 
-/*
+ /*
  * @test
- * @bug 8039951
+ * @bug 8039951 8281717
```

Comment:
No need to add bug id here as this is not a product change.

Reply:
Done.